src/client/src/App.tsx

```diff
@@ -35,6 +35,9 @@ const ViewLicensesModal = loadable(
 const AzureServicesModal = loadable(
   () => import(/* webpackChunkName: "AzureServicesModal" */ "./modals/AzureServicesModal")
 );
+const ViewPlatformRequirementsModal = loadable(
```

Comment:
We could remove "View" from the name to shorten it a bit I think.

Reply:
Done.